lib/dkim.js

```diff
@@ -463,8 +463,9 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = err.message;
+
+        return callback(null, sigs); // returning the error AND the sigs breaks promisification... should do callback(null, sigs)
```

Comment:
Is there _any_ error that `keyFromDNS` might return that we wouldn't want to swallow? How would a totally random nodejs error look in the dkim validator UI, do we know? I forget how these kind of errors would manifest.